src/Service/NotificationService.php

```diff
@@ -94,35 +84,76 @@ public function __construct(
      */
     public function notificationHandler(array $data, array $configuration): array
     {
-        if ($data['method'] !== 'POST') {
```

Comment:
I think we do want this here and else we want this to be configurable using $configuration.
$configuration['allowedmethods'] = [post,put]
in_array('method', $configuration['allowedmethods'])